entrypoint.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash 
+###########################################################################################
+# This script pulls the CA bundle from the host wireserver, parses it,                    #
+# and installs them in the Ubuntu CA bundle. Recommended execution method                 #
+# is cloud-init or VM custom-data for execution at provisioning time                      #
```

Comment:
Maybe add a note here saying this is required for specialized sovereign clouds where CA bundle is completely different from normal production clouds?